js/testing.js

```diff
@@ -0,0 +1,61 @@
+// Testing.js
+
+(function(){
+
+	var newTestingDiv;
+
+function test_group(name, _callback){
+
+	var testingDiv = document.getElementById("testingDiv");
+
+	newTestingDiv = document.createElement("div");
+	newTestingDiv.className = "row col-md-12";
+	newTestingDiv.style.marginTop = "5%";
+	testingDiv.appendChild(newTestingDiv);
+
+	_callback();
+
+	newTestingDiv.childNodes.forEach(function(currAssert){
+		if (currAssert.style.backgroundColor === 'red'){
+			newTestingDiv.style.backgroundColor = 'red';
+		}
+		else if (newTestingDiv.style.backgroundColor != 'red'){
+			newTestingDiv.style.backgroundColor = 'green';
+		}
+	});
+}
+
+function assert(condition, message) {
```

Comment:
Looks like a very specific assert for a specific assert. Not a genetic assert at all..

Reply:
We don't need to test something, just create an element for a test.